src/utilities/getShortcut.ts

```diff
@@ -0,0 +1,29 @@
+import { registerCommand, executeCommand } from "../utilities/getRandom";
+import { SHORT_CUT } from "../const";
+import * as os from "os";
+
+export function getRandomShortcut() {
+  registerCommand("getRandomShortcut", SHORT_CUT);
+  const shortcut = executeCommand("getRandomShortcut");
+  const platform = os.platform();
+  let message;
+  let ShortcutProp;
+  if (platform === "darwin") {
+    message = `${shortcut.name}: ${shortcut.darwin?.command}`;
+    ShortcutProp = {
+      name: shortcut.name,
+      description: shortcut.description,
+      command: shortcut.darwin?.command,
+      gif: shortcut.darwin?.gif,
+    };
+  } else {
+    message = `${shortcut.name}: ${shortcut.win32?.command}`;
+    ShortcutProp = {
+      name: shortcut.name,
+      description: shortcut.description,
+      command: shortcut.win32?.command,
+      gif: shortcut.win32?.gif,
+    };
+  }
```

Comment:
ここはif/elseだったりshortcut.wind32などが冗長に見えますね🤔
下記のようにすればシンプルにできそうです！

また、"darwin" なども const に切り出して、`OS.darwin` のように記述できるとさらに型安全で良さそうですね👀
```suggestion
    const platformShortcut = platform === "darwin" ? shortcut.darwin : shortcut.win32;

    const message = `${shortcut.name}: ${platformShortcut?.command}`;

    const shortcut = {
    name: shortcut.name,
    description: shortcut.description,
    command: platformShortcut?.command,
    gif: platformShortcut?.gif,
    };
```